audio-worklet/src/noisegate-script-processor.js

```diff
@@ -98,7 +106,7 @@ class NoiseGate {
   onaudioprocess_(event) {
     let inputBuffer = event.inputBuffer;
     let channel0 = inputBuffer.getChannelData(0);
-
+    
```

Comment:
I'm confused about what you want me to remove here. When I look at the file here, there is a blank line in between `let channel0 = inputBuffer.getChannelData(0); ` and the comment beneath it. Do you want me to remove the blank line? I think the blank line here is warranted since it's separating to logically distinct units of the code, and the next line is a comment.